### PR TITLE
[BTAT-9210] New feature switch for dd interrupt

### PR DESCRIPTION
**Pull request by daparkes**

### CODE REVIEW

**Comment by daparkes** [comment]
app config pr: https://github.com/hmrc/app-config-production/pull/11191

File: conf/application.conf [hunk 1]
```diff
@@ -138,6 +138,7 @@ features {
   displayCovid.enabled = true
   missingTraderAddressIntercept.enabled = true
   r17Content.enabled = true
+  directDebitInterrupt.enabled = true
```
Comment (lewishay) [inline conf/application.conf:141]: Can you please also add the feature switch elements for this to allow it to be toggled via the feature switch test-only page

File: app/config/ConfigKeys.scala [hunk 2]
```diff
@@ -40,6 +40,7 @@ object ConfigKeys {
   val displayCovidMessageFeature: String = "features.displayCovid.enabled"
   val missingTraderAddressIntercept: String = "features.missingTraderAddressIntercept.enabled"
   val r17Content: String = "features.r17Content.enabled"
+  val directDebitInterrupt: String = "features.ddInterrupt.enabled"
```
Comment (lewishay) [inline app/config/ConfigKeys.scala:43]: This String needs to match up with the String you have in `application.conf`

**Review by lewishay — CHANGES_REQUESTED** [review]

**Review by lewishay — APPROVED** [review]
Approved awaiting clear pipeline

**Pull request merged**